apps/assisted-disconnected-ui/Containerfile.ocp

```diff
@@ -1,23 +1,43 @@
-FROM registry.ci.openshift.org/ocp/builder:rhel-9-base-nodejs-openshift-4.19 AS ui-build
-USER root
+FROM registry.ci.openshift.org/ocp/builder:rhel-9-base-nodejs-openshift-4.20 AS ui-build
+USER 0
 
 WORKDIR /app
-COPY --chown=1001:0 / /app
-RUN ls /app
+COPY --chown=1001:0 . /app
 ENV NODE_OPTIONS='--max-old-space-size=8192'
 RUN git config --global --add safe.directory /app
-RUN npm install -g corepack@0.24.1
-RUN yarn install --immutable && yarn build:all
 
-FROM registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.23-openshift-4.19 as proxy-build
+# The REMOTE_SOURCES value is set by the build system to indicate the location of the cachito-backed artifacts cache.
+COPY $REMOTE_SOURCES $REMOTE_SOURCES_DIR
+
+ARG YARN_VERSION=3.4.1
+
+# bootstrap yarn so we can install and run the other tools.
+RUN CACHED_YARN=./artifacts/${YARN_VERSION}.tar.gz; \
+    if [ -f ${CACHED_YARN} ]; then \
+      # use dependencies provided by Cachito
+      npm install -g ${CACHED_YARN}; \
```

Comment:
@rawagner Is this correct and sufficient to install yarn 3.4.1 or it still needs corepack?

Reply:
installing only yarn 3.4.1 should be enough